_data/events/20200602-virtual-pipelines-training.yml

```diff
@@ -0,0 +1,7 @@
+name: Virtual Pipelines Training
+startdate: 2019-06-2
+enddate: 2019-06-4
```

Comment:
looks like the CI checks fail - likely because you need 0s on the dates.

Reply:
```suggestion
enddate: 2019-06-04
```